terraform/environments/mlra/modules/ecs/main.tf

```diff
@@ -136,7 +138,7 @@ resource "aws_launch_template" "ec2-launch-template" {
 
   metadata_options {
     http_endpoint               = "enabled"
-    http_tokens                 = "optional"
+    http_tokens                 = "required"
```

Comment:
Can we be sure this will not break anything? It was flexible before and is now required.

Reply:
Hmmm yeah fair point, have reverted, we can revisit as part of LASB-3390.